addons/event_booth_sale/static/src/js/event_booth_configurator_widget.js

```diff
@@ -0,0 +1,148 @@
+odoo.define('event_booth_sale.product_configurator', function (require) {
+const ProductConfiguratorWidget = require('sale.product_configurator');
+
+/**
+ * Extension of the ProductConfiguratorWidget to support event booth configuration.
+ * It opens when an event booth product_product is set.
+ *
+ * The event booth information include:
+ * - event_id
+ * - event_booth_id
```

Comment:
Is that really all of it?
Seems like you pass more info when looking at the various default values.